swagger/v1/openapi.json

```diff
@@ -291,7 +423,7 @@
                   "$ref": "#/definitions/links"
                 },
                 "data": {
-                  "type": "array",
+                  "type": "object",
```

Comment:
I think this "array" object is correct, and what was required was to specify the "type: object" for each of the elements in the array : https://github.com/RedHatInsights/compliance-backend/pull/250/files#diff-bbc70b65028c3e55922508735e913f6eR275

In addition, can we update the openapi.json to be OpenAPI ver. 3 ? ( and keep the .json instead of .yaml file ? )

Reply:
The current rswag gem does not support that yet, so we have to make the conversion to v3 and JSON manually. However, there is a fork we might be able to use for the time being https://github.com/jdanielian/open-api-rswag

Reply:
okay, anything that helps in the meantime to get an updated `openapi V3` file will work I guess.